docs/installation-guide.md

```diff
@@ -28,7 +28,7 @@ It is okay to install ADF and AWS Control Tower in different regions.
 For example:
 
 - Install AWS Control Tower in `eu-central-1`.
-- Install ADF in `us-east-1`.
+- Install ADF in `us-east-1` or `cn-north-1`.
```

Comment:
Does that work, having AWS Control Tower deployed in `eu-central-1` while deploying ADF in `cn-north-1`?